ideascube/migrations/0009_add_a_system_user.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from ideascube.models import User
+
+
+def add_user(*args):
+    User(serial='__system__', full_name='System', password='!!').save()
```

Comment:
Incidentally, this is exactly how this works on Linux. :)

```
# grep mathieu /etc/shadow
mathieu:the_hash_of_my_password:16988:0:99999:7:::
# useradd tmptest
# grep tmptest /etc/shadow
tmptest:!!:17032:0:99999:7:::
```

That doesn't mean it is a good idea, but I figured I would at least mention where I took the idea from.